lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,33 @@
+<div class="create-team wrapper-msg is-incontext urgency-low warning is-hidden">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%- gettext("Oops!") %></h3>
```

Comment:
@catong yes, the heading message will apply to all error messages.
so now the Oops will be replaced by `Your team could not created` and 
`Your team could not be created. Check the highlighted fields below and try again.` message remain as is. right?